pulp/tests/test_pulp.py

```diff
@@ -356,7 +357,7 @@ def test_pulp_030(self):
         prob += -y + z == 7.5, "c3"
         self.solver.mip = 0
         print("\t Testing MIP relaxation")
-        if self.solver.__class__ in [GUROBI_CMD, PULP_CHOCO_CMD, CHOCO_CMD, MIPCL_CMD]:
+        if self.solver.__class__ in [GUROBI_CMD, PULP_CHOCO_CMD, CHOCO_CMD, MIPCL_CMD, SCIP_CMD]:
```

Comment:
SCIP has no option to relax the problem ? or is it just not part of the current interface? I ask in order to add it as a TODO somewhere in case it's just missing from the SCIP_CMD interface.

Reply:
I think you can relax problems with SCIP by adding a relaxation handler? However, the current interface here doesn't seem to be handling it. Though I don't know much about this in general so I could be wrong.